e2e/cypress/integration/ui/settings/team_mgmt.spec.ts

```diff
@@ -105,8 +105,7 @@ describe('team management', () => {
 
       cy.get('[data-cy=save-button]').click();
       cy.get('app-team-management chef-modal').should('not.be.visible');
-      cy.get('#main-content-wrapper').scrollTo('top');
```

Comment:
now that notifications do not push content down, many of these tests were failing because the page is no longer long enough to be scrolled.  Several of these `scrollTo` have been removed to remedy the problem.